### ISSUE TITLE
exhaustMapWithTrailing not work with EMPTY, can you add support for it?

### ISSUE THREAD

**Issue by zdu-strong** [bjoerge/rxjs-exhaustmap-with-trailing#5]
I have this code.

    range(1, 30).pipe(
        exhaustMapWithTrailing((s) => {
            if(s < 15){
                return EMPTY;
            }
            return timer(1000).pipe(mapTo(s));
        }),
        tap((s) => {
            console.log(s);
        }),
    )
    
   It didn't print anything.
   Actually, replace EMPTY with of(s) and it will work just fine.

**Comment by bjoerge** [comment]
I believe this is a duplicate of https://github.com/bjoerge/rxjs-exhaustmap-with-trailing/issues/4

Forcing it to be async should work:
```js
range(1, 30).pipe(
  exhaustMapWithTrailing((s) => {
    if (s < 15) {
      return scheduled(EMPTY, asyncScheduler);
    }
    return timer(1000).pipe(mapTo(s));
  }),
  tap((s) => {
    console.log(s);
  })
);
```

**Comment by bjoerge** [comment]
Fixed & released in [v2.1.0](https://github.com/bjoerge/rxjs-exhaustmap-with-trailing/commit/57c492a53008209ee1d609f26ca3aaf5757d95b0) and [v1.2.0](https://github.com/bjoerge/rxjs-exhaustmap-with-trailing/releases/tag/v1.2.0)
